docs/root/api-v1/http_filters/gzip_filter.rst

```diff
@@ -0,0 +1,70 @@
+.. _config_http_filters_gzip_v1:
```

Comment:
I can't remember if we already discussed this, but do we need v1 support for this? If not, prefer to drop it, since we will soon be deprecating v1.